tailwind.config.ts

```diff
@@ -63,9 +49,6 @@ export default {
       textColor: {
         ...textColorTokens,
       },
-      spacing: {
-        ...sizeTokens,
-      },
```

Comment:
I forgot about this part 😅 This is good to remove, but we also want to bring in the tokens that have `-spacing-` in their instead ([example](https://github.com/chanzuckerberg/edu-design-system/pull/2156/files#diff-ae098022f6228c869ce589a6d9c92799310df9bb7b6533d95dca208a5fc32f51L29))

What we want to spread here is all of the `eds-spacing-size` values that get generated, so tailwind can have stuff like `px-spacing-size-1` or similar

(& in the migration script, we will add a migration that changes `X-size-3` to `X-spacing-size-3` in consuming projects, for reference)